From dfc1e8a70a5ee1b35c9e98edb9687fff97eb6012 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Christoph=20Gr=C3=BCninger?= <foss@grueninger.de>
Date: Wed, 28 May 2025 20:22:29 +0200
Subject: [PATCH] Remove comment which triggers a warning

configure.ac:52: warning: whitespace-separated list contains macros;
configure.ac:52: in a future version of Autoconf they will not be expanded
configure.ac:52: note: 'dnl' is a macro
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 1afae42..f043163 100644
--- a/configure.ac
+++ b/configure.ac
@@ -50,7 +50,7 @@ AC_SUBST([REVENGE_LIBS])
 # Find boost
 # ==========
 AC_CHECK_HEADERS(
-  [dnl
+  [
     boost/cstdint.hpp \
     boost/math/constants/constants.hpp \
     boost/optional.hpp \
-- 
2.51.0

